rewrite-maven/src/test/java/org/openrewrite/maven/ChangeParentPomTest.java

```diff
@@ -236,6 +236,181 @@ void changeParentAddRelativePathNonEmptyValue() {
         );
     }
 
+    @Test
+    @RepeatedTest(10)
```

Comment:
There's no need to tag a test with both `@Test` and `@RepeatedTest`; I'm assuming you wanted to retain the `@RepeatedTest`, but we can also go for `@Test` here.
```suggestion
    @RepeatedTest(10)
```